scripts/genpinctrl/config.yaml

```diff
@@ -0,0 +1,139 @@
+# Configuration file for pinctrl generation (except F1 series).
+#
+# This file contains a list of pin configuration templates used to generate the
+# pinctrl files. Each entry can have the following fields:
+#
+#   - name (mandatory): This is the pin alternate function name, e.g. UART_TX.
+#     It is used to group pin configurations alphabetically in the generated
+#     pinctrl files.
+#
+#   - match (mandatory): This is a regular expression used to match against
+#     CubeMX pin configuration names. The regular expression should be as
+#     precise as possible. Note that it needs to be escaped here in the
+#     configuration file.
+#
+#   - bias (optional): Bias setting (disable, pull-up, pull-down). If not set
+#     pinctrl file will not contain the bias entry, equivalent to disable
+#     setting.
+#
+#   - drive (optional): Drive setting (push-pull, open-drain). If not set
+#     pinctrl file will not contain the drive entry, equivalent to push-pull
+#     setting
+#
+#   - variant (optional): Define another variant of a pin configuration. This
+#     setting can be useful to define multiple configurations for a certain
+#     alternate function.
+#
+
+---
+- name: ADC_IN / ADC_INN / ADC_INP
+  match: "^ADC(?:\\d+)?_IN[NP]?\\d+$"
+
+- name: CAN_RX
+  match: "^CAN\\d+_RX$"
+  bias: pull-up
+
+- name: CAN_TX
+  match: "^CAN\\d+_TX$"
+
+- name: DAC_OUT
+  match: "^DAC(?:\\d+)?_OUT\\d+$"
+
+- name: FDCAN_RX
+  match: "^FDCAN\\d+_RX$"
+  bias: pull-up
```

Comment:
Current code is `STM32_PUSHPULL_NOPULL`

Reply:
Should `CAN_RX` bias be removed then?

Reply:
for CAN_RX, we have both PULLUP and NOPULL in the code right now.
Let's see what users report during review.